### DISCUSSION TITLE
Override mount point from template?

### DISCUSSION THREAD

**Discussion by lattice0** [lima-vm/lima#1845] [Q&A]
```
limactl start \
  --name=$MACHINE_NAME \
  --cpus=$CPU_COUNT \
  --memory=$MEMORY \
  --tty=false \
  template://ubuntu
```

will mount the folder ˜ from the template. Is there a way to remove this via the command line? I tried passing `-mounts=[]` and other variations but it didn' t work. I didn' t want to create a file just for removing the disk, I want everything to be in a command for ease of usage in a script

**Comment by jandubois** [comment]
Use `--set '.mounts=[]'`.

**Reply by lattice0** [reply]
I updated to th enewest version and got ˋFATA[0000] field `mounts[0].location` is internally reservedˋ

**Reply by afbjorklund** [reply]
You need to change "mountpoint", not the "location".

```yaml
  # Configure the mountPoint inside the guest.
  # 🟢 Builtin default: value of location
  mountPoint: null

```

But there should **not** be any errors from empty mounts?

`--set '.mounts=[]'`

e.g.
```yaml
# Mounts are disabled in this template, but can be enabled optionally.
mounts: []

```

**Reply by lattice0** [reply]
Somehow doing ˋ  --set '.video.display="vnc"' \
  --set '.mounts=[]' \ˋ works, while   --set '.mounts=[]' \  --set '.video.display="vnc"' \ gives that error

**Reply by lattice0** [reply]
hmmm, it looks like I was using it inside another lima VM
